esp32/modules/boot.py

```diff
@@ -38,15 +38,18 @@
         ugfx.clear(ugfx.WHITE)
         ugfx.flush(ugfx.LUT_FULL)
 except BaseException as e:
+    sys.print_exception(e)
+    import easydraw
+    easydraw.msg("A fatal error occured!","Still Crashing Anyway", True)
+    easydraw.msg("")
```

Comment:
The exception message is reported on line 51/54. It's the diff-tool which erroneously displays these lines as added.
(I think this discussion is outside the scope of this patch)